dotnet/src/Functions/Functions.OpenApi/OpenApi/OpenApiDocumentParser.cs

```diff
@@ -237,6 +237,43 @@ internal static List<RestApiOperation> CreateRestApiOperations(OpenApiDocument d
         }
     }
 
+    /// <summary>
+    /// Returns a list of parameters that are defined in the path and operation.
+    /// Will deduplicate operation parameters overrides.
+    /// </summary>
+    /// <param name="pathItem">The path item.</param>
+    /// <param name="operation">The operation.</param>
+    /// <returns>The list of parameters.</returns>
+    private static IEnumerable<OpenApiParameter> GetDeduplicatedParametersOverrides(OpenApiPathItem pathItem, OpenApiOperation operation)
+    {
+        return pathItem.Parameters
```

Comment:
Can we replace seven LINQ operations with this one: `return operation.Parameters.Union(pathItem.Parameters, s_parameterNameAndLocationComparer);`?

Reply:
great suggestion! after fixing the string interpolation, I didn't realize I could clean it up further!